contracts/v0.8/utils/Errors.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.18;
+
+library Errors {
```

Comment:
Shouldn't this util be defined in `tests` folder if it's only used in tests?

Reply:
It is used in tests only, but we want to give the developer that uses this lib a reference way to process the exit codes, and so it's in `utils` folder instead of the `tests`.